labs/preparation/missing_values_services.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+
+from helpers.dslabs_functions import mvi_by_filling
+
+
+def imputate_missing_values_services():
+    filename = '../../datasets/prepared/class_credit_score_encoded_1.csv'
+    df = pd.read_csv(filename, na_values="")
+
+    try:
+        df.drop(columns=["Customer_ID"], inplace=True)
+    except KeyError:
+        pass
+
+    knn = mvi_by_filling(df, 'knn')
+    knn.to_csv('../../datasets/prepared/class_credit_score_encoded_2_MVI_knn.csv', index=False)
+
+    freq = mvi_by_filling(df, 'frequent')
```

Comment:
My idea is to remove rows and columns that has >90% or >80%  missing values and then use Median or Mode.